wafer/talks/tests/test_talk_type.py

```diff
@@ -0,0 +1,54 @@
+def test_open_by_default():
+    from wafer.talks.models import TalkType
```

Comment:
I assume this strange importing of things inside the test functions was copied from `test_wafer_basic_talks.py`? If so, does anyone remember why it is like that? If not, perhaps we should just try importing things the ordinary way at the top of the module?

Reply:
This pattern is from the first tests we wrote  - I think we needed to work around some odd behaviour with how importing `django.contrib.auth` worked when running the tests at the time, and just never updated it once that was no longer needed.

Reply:
ack, I moved the imports to the top